recipes/cppimport/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "cppimport" %}
+{% set version = "22.8.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: bbb4957102db41bc99ad72c233bce92f9d1fd91be352fc07878c4361033a401f
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+  host:
+    - python
```

Comment:
Should be the same as run for noarch. Sorry I missed it before clicking on merge :-/

It is not a big deal but it would be nice to fix in the feedstock.